app/presenters/publishing_api/document_collection_presenter.rb

```diff
@@ -0,0 +1,72 @@
+module PublishingApi
+  class DocumentCollectionPresenter
+    include UpdateTypeHelper
+
+    attr_reader :update_type
+
+    def initialize(item, update_type: nil)
+      @item = item
+      @update_type = update_type || default_update_type(item)
+    end
+
+    def content_id
+      item.content_id
+    end
+
+    def content
+      content = BaseItemPresenter.new(item).base_attributes
+      content.merge!(
+        description: item.summary,
+        details: details,
+        document_type: "document_collection",
+        public_updated_at: item.public_timestamp || item.updated_at,
+        rendering_app: Whitehall::RenderingApp::WHITEHALL_FRONTEND,
+        schema_name: "document_collection",
+      )
+      content.merge!(PayloadBuilder::PublicDocumentPath.for(item))
+    end
+
+    def links
+      links = LinksPresenter.new(item).extract(
+        %i(organisations policy_areas topics related_policies)
+      )
+      links.merge!(documents: item.documents.map(&:content_id))
+      links.merge!(PayloadBuilder::TopicalEvents.for(item))
+    end
+
+  private
+
+    attr_reader :item
+
+    def details
+      {
+        first_public_at: first_public_at,
+        change_history: item.change_history.as_json,
+        collection_groups: collection_groups,
+        body: govspeak_renderer.govspeak_edition_to_html(item),
+        emphasised_organisations: item.lead_organisations.map(&:content_id),
+      }.tap do |details_hash|
+        details_hash.merge!(PayloadBuilder::PoliticalDetails.for(item))
+        details_hash.merge!(PayloadBuilder::AccessLimitation.for(item))
+      end
+    end
+
+    def first_public_at
+      item.first_published_at || item.document.created_at
```

Comment:
We've decided to leave this for now as `first_public_at` is already exposed on Edition with different functionality and is used elsewhere. There are cards in Trello https://trello.com/c/j7B7RSMa/729-migrated-formats-no-longer-need-to-send-first-public-at and https://trello.com/c/YhI80ytY/727-ensure-migrated-formats-are-rendered-using-first-published-at to address this.